packages/create-hint/src/no-official-files/.eslintrc.json

```diff
@@ -258,7 +255,7 @@
         "typescript/no-namespace": "error",
         "typescript/no-parameter-properties": "error",
         "typescript/no-triple-slash-reference": "error",
-        "typescript/no-unused-vars": "error",
+        "typescript/no-unused-vars": "off",
```

Comment:
If we're updating the templates, it seems we should also turn on the `noUnusedLocals` compiler flag in `shared-templates/tsconfig.json.hbs`.

Reply:
I was sure you were going to say this 😁
I think I'll enable it on the path that's not `official`. The reason is that we are extending from the main `tsconfig.json`. And you will say:
- "But we have `strict: strue`! 
- Yes, but the other packages also have it and it was for consistency that we added it.

One thing I've realized is that we don't have `strict: true` in the `non-official` path though..

Does this sound ok?